### PR TITLE
Add a recipe for MPAS-Analysis

### PR DESCRIPTION
**Pull request by xylar**
Add a recipe for [MPAS-Analysis](https://github.com/MPAS-Dev/MPAS-Analysis)

### CODE REVIEW

**Comment by xylar** [comment]
@jhkennedy, please confirm that you're okay with being a maintainer of the `mpas-analysis` feedstock.

**Comment by jhkennedy** [comment]
Yep, happy to be a maintainer!

**Comment by xylar** [comment]
I wonder if the Windows error is a docker problem with symlinks, related to:
https://github.com/docker/compose/issues/3902

**Comment by xylar** [comment]
@conda-forge/help-python, I could use some assistance understanding why my symlinks aren't copying correctly in appveyor. (Here and in #7836.)

File: recipes/mpas-analysis/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/MPAS-Dev/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 9a9c99c8a6c510d2f552a88744095ae31c67cb07f7b519d9ba6f6543f4584406
+
+build:
+    number: 0
+    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+    noarch: python
+    entry_points:
+        - mpas_analysis = mpas_analysis.__main__:main
```
Comment (chrisburr) [inline recipes/mpas-analysis/meta.yaml]: Before I forget, there is a second entry_point in the `setup.py` called `download_analysis_data`.

**Comment by chrisburr** [comment]
[This](https://github.com/conda-forge/conda-smithy/issues/1002#issuecomment-454856614) summarises the issue (points 3 and 4 are only relevant for fixing that manifestation of the issue).

I see two possible solutions:

1. Ignore the failure, this is a `noarch` package so it will be built in circle by the actual feedstock. I'm not sure what will happen when installing but the affected files only(?) needed for running the tests so it doesn't really matter.
2. Replace the symlinks with copies using something like (again this will only work when building on unix but it's a `noarch` package):
```bash
find . -type l -exec bash -c 'fn="{}" && linked_fn="$(readlink "${fn}")" && rm "${fn}" && cp -p "${linked_fn}" "${fn}"' \;
```

File: recipes/mpas-analysis/meta.yaml (hunk 1 above)
Comment (xylar) [reply recipes/mpas-analysis/meta.yaml]: There sure is, and it's missing from my own conda recipe, meaning the packages on my channel are broken. Thank you for noticing that.

**Comment by xylar** [comment]
Okay, great!  I'm glad to know it's a known issue.

I'm very much inclined toward 1 above.  We've never had any Windows users of this package before so it's an added benefit if it happens to work out but wouldn't be a big loss for us if it doesn't. 

Again, much appreciated.

**Comment by xylar** [comment]
Again, thanks so much!

**Pull request merged**